config.ts

```diff
@@ -32,66 +27,52 @@ export interface DisplayedField {
   precision?: number;
 }
 
-interface ContractConfig {
-  JUNIOR_OPERATOR: 'ALLOWANCE_OPERATOR';
-  SENIOR_OPERATOR: 'PROPORTIONAL_OPERATOR';
+interface Config {
+  rpcUrl: string;
+  etherscanUrl: string;
+  gasLimit: number;
+  transactionTimeout: number;
+  tinlakeDataBackendUrl: string;
+  isDemo: boolean;
+  network: 'Mainnet' | 'Kovan';
+  pools: Pool[];
 }
 
 const contractAddressesSchema = yup.object().required().shape({
-  DEPLOYMENT_NAME: yup.string().required(),
   ROOT_CONTRACT: yup.string().length(42).matches(/0x[0-9a-fA-F]{40}/).required(),
-  TINLAKE_CURRENCY: yup.string().length(42).matches(/0x[0-9a-fA-F]{40}/).required(),
   ACTIONS: yup.string().length(42).matches(/0x[0-9a-fA-F]{40}/).required(),
   PROXY_REGISTRY: yup.string().length(42).matches(/0x[0-9a-fA-F]{40}/).required(),
-  COLLATERAL_NFT: yup.string().length(42).matches(/0x[0-9a-fA-F]{40}/).required()
-});
-
-const nftDataDefinitionSchema = yup.object().required().shape({
-  contractCall: yup.object().required().shape({
-    outputs: yup.array(yup.object().shape({
-      name: yup.string().required().min(1),
-      type: yup.mixed<'address' | 'uint256' | 'string'>().required().oneOf(['address', 'uint256', 'string'])
-    }))
-  }),
-  displayedFields: yup.array(yup.object().shape({
-    key: yup.string().required().min(1),
-    label: yup.string().required().min(1),
-    type: yup.string().required().min(1),
-    decimals: yup.number().min(0),
-    precision: yup.number().min(0)
-  }))
+  COLLATERAL_NFT: yup.string().length(42).matches(/0x[0-9a-fA-F]{40}/),
```

Comment:
Is this address not required? If not, we should change the type to `string?` in line 9.

Reply:
not required. Just needed for testing purposes & the old demo setup.